node/pkg/terra/watcher.go

```diff
@@ -115,21 +136,21 @@ func (e *Watcher) Run(ctx context.Context) error {
 	}
 	err = c.WriteJSON(command)
 	if err != nil {
-		p2p.DefaultRegistry.AddErrorCount(vaa.ChainIDTerra, 1)
+		p2p.DefaultRegistry.AddErrorCount(e.chainID, 1)
 		terraConnectionErrors.WithLabelValues("websocket_subscription_error").Inc()
 		return fmt.Errorf("websocket subscription failed: %w", err)
 	}
 
 	// Wait for the success response
 	_, _, err = c.ReadMessage()
 	if err != nil {
-		p2p.DefaultRegistry.AddErrorCount(vaa.ChainIDTerra, 1)
+		p2p.DefaultRegistry.AddErrorCount(e.chainID, 1)
 		terraConnectionErrors.WithLabelValues("event_subscription_error").Inc()
 		return fmt.Errorf("event subscription failed: %w", err)
 	}
 	logger.Info("subscribed to new transaction events")
```

Comment:
In the Eth watcher, we have a networkName which is included in all the log messages so we can tell which chain the error is coming from. Do we want to start doing that for Terra now?

Reply:
Yes, yes we absolutely do!

Reply:
thanks! added `terra_network` to the metric (possibly to be renamed in the future)